src/parsy/__init__.py

```diff
@@ -194,7 +213,7 @@ def alt_parser(stream, index):
 
     return alt_parser
 
-def seq(*parsers):
+def seq(*parsers, f=None):
```

Comment:
Not up to me, but I'd prefer f be named seq_fn for consistency with map/bind.

Reply:
I think that this extra keyword parameter would work better as a separate method, perhaps named `combine`. I can see situations where it would be useful outside of the context of `seq`, such as after `many`, or if you want to reuse a parser defined elsewhere, and it would be cleaner this way I think.

Reply:
I may not be following; how would it be different from `.map()`?

And my present self is not that convinced with "compare before and after" example from my 2.5-year-younger self, so maybe we don't need that feature at all.

Reply:
I've added a PR for this feature here - https://github.com/python-parsy/parsy/pull/3 - which happens to include docs that answer your question. Let me know what you think.

Reply:
Oh. Yes, that looks good and is indeed cleaner. :+1: 

How would you use it with `.many()` though? (you don't know the arg count; you use `.times()` when you do know)

Reply:
You're right, `times()` would be a better example, although occasionally you might use `manyy()` followed by some constructor functions that take an arbitrary number of arguments via`*args`.